apps/platform/src/components/dashboard/overview/ServerKeySetup/index.tsx

```diff
@@ -48,23 +48,12 @@ function ServerKeySetup({
       onOpenStoreDialog()
     }
   }
+  const isPrivateKeyStored = privateKey && isStoredOnServer
```

Comment:
I think we can move this to an atom. This is used in many places. Perhaps we can use something like this:
`type privateKeyStorageType = 'IN_DB' | 'IN_BROWSER' | 'ABSENT'`